scripts/release.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+set -e
+
+info() {
+    echo -e "\033[1;34m$1\033[0m"
+}
+
+step() {
+    echo ""
+    info "$1"
+}
+
+info "Starting the release process"
+
+step "make install"
+make install
+
+step "make build"
+make build
+
+step "manual tests: Run the EE tests"
+echo "Copy the the packages folder content inside the node_modules of ra-enterprise, then run a full build and run the tests"
+echo "Tip: You can use the 'copy-ra-oss-packages-to-ee.sh' script if you have it"
+echo "Press Enter when this is done"
+read
```

Comment:
I think we should agree on a directory structure and do this automatically

Reply:
yes I have plans for this, but will do that in a later PR (hopefully)